dashboard/controller/assignments/exportCSV.js

```diff
@@ -0,0 +1,92 @@
+// include libraries
+var superagent = require('superagent'),
+	async = require('async'),
+	common = require('../../helper/common');
+
+module.exports =  function exportCSV(req, res) {
+	var assignment = [];
+
+	const validateAssignment=(assignment)=>{
+		const assign={
+			_id:"",
+			name:"",
+			title:"",
+			activity:"",
+			creation_time:"",
+			stroke:"",
+			fill:"",
```

Comment:
The fill and stroke information is not relevant to delivery.

Reply:
sure